recipes/hwdata/all/conanfile.py

```diff
@@ -0,0 +1,84 @@
+from conan import ConanFile
+from conan.tools.env import VirtualBuildEnv
+from conan.tools.files import copy, get, rmdir
+from conan.tools.gnu import Autotools, AutotoolsToolchain
+from conan.tools.layout import basic_layout
+import os
+
+
+required_conan_version = ">=1.53.0"
+
+
+class HwDataConan(ConanFile):
+    name = "hwdata"
+    description = "hwdata contains various hardware identification and configuration data, such as the pci.ids and usb.ids databases"
+    license = "GPL-2.0-or-later OR XFree86-1.1"
```

Comment:
```suggestion
    license = ("GPL-2.0-or-later", "XFree86-1.1")
```
Each license should a separated item, so we can filter them by conan inspect command

Reply:
Great catch

Reply:
No problem, but I do get the feeling will want to distinguish between `AND` and `OR` in the future. No need to have that discussion here though. I'll update that right now.

Reply:
Fixed.

Reply:
I agree, thanks for the heads up